src/content/getusermedia/audio/js/main.js

```diff
@@ -9,13 +9,11 @@
 'use strict';
 
 // Put variables in global scope to make them available to the browser console.
-var audio = window.audio = document.querySelector('audio');
+var audio = document.getElementById('gum-local');
```

Comment:
I think if you need to use `querySelector()` once, it's neater and slightly more understandable to use it for everything.

`getElementById()` is quicker, but that's not an issue here.

Don't feel very strongly though.